### ISSUE TITLE
Uncaught TypeError: sx.prompt is not a function

### ISSUE THREAD

**Issue by fonclub** [skeeks-semenov/yii2-ajax-file-upload#3]
Привет. При клике по кнопке _Загрузить по ссылке_ в js консоли ошибка **tool-remote-upload.js:19 Uncaught TypeError: sx.prompt is not a function**

**Comment by skeeks-semenov** [comment]
Привет. Если есть возможность, можно потестировать. В версии 1.0.4 поправил.
https://github.com/skeeks-semenov/yii2-ajax-file-upload/releases/tag/1.0.4

**Comment by fonclub** [comment]
Да, теперь все четко, спасибо! Один вопрос есть - при переопределении в конфиге виджета значения 'tools' => [], если указать только 
`'default' =>
                    [
                        'class' => AjaxFileUploadDefaultTool::class,
                        'name'  => 'Загрузить',
                        'icon'  => 'glyphicon glyphicon-download-alt',
                    ],`
кнопки не отображаются вообще. Я пытался вывести только кнопку для выбора файла, без возможности загрузки по ссылке, но что-то не срослось)
